OpenRA.Game/Traits/World/Faction.cs

```diff
@@ -29,6 +30,8 @@ public class FactionInfo : TraitInfo<Faction>
 		[Desc("The side that the faction belongs to. For example, England belongs to the 'Allies' side.")]
 		public readonly string Side = null;
 
+		[TranslationReference(optional: true)]
```

Comment:
you made these optional but don't handle them being null. Is that something to be concerned about?

Reply:
Yes, it hides a bug where the linter spits out
> OpenRA.Utility(1,1): Warning: Empty key in mod translation files required by Actor `world` trait `Faction.Description`
OpenRA.Utility(1,1): Warning: Empty key in mod translation files required by Actor `editorworld` trait `Faction.Description`

Reply:
I added more null checks and made the `Name` not optional.